### DISCUSSION TITLE
Multi session telnet

### DISCUSSION THREAD

**Discussion by Hsubtnarg** [sstaub/Ethernet3#60] [General]
Thanks for creating this library it is a great improvement, especially setHostname().

I am trying to create a Telnet server that can multiple sessions open at once, as shown in the Arduino Reference for Ethernet

https://reference.arduino.cc/reference/en/libraries/ethernet/server.accept/

But the accept() function in your library is not public and does not return a pointer to EthernetClient as the original Ehternet library did. Do you have any plans enable this or is there another way I can create a multi session telnet server.

Thanks

**Comment by sstaub** [comment]
In the moment I don't have time to maintain this library. Maybe you use the Arduino library instead https://github.com/arduino-libraries/Ethernet
Over the years nearly all functions of this library have were also included in the official library. Only the begin() function is different,
other functions have only a different name.
